04/style/style.css

```diff
@@ -0,0 +1,157 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-family: Arial, sans-serif;
+    background-color: whitesmoke;
+}
+
+
+.hamburger {
+    width: 40px;
+    height: 35px;
+    border: 0;
+    background: silver;
+    position: relative;
+    cursor: pointer;
+    display: inline-block;
+}
+
+.hamburger::before, .hamburger::after {
+    content: '';
+    position: absolute;
+    left: 0;
+    width: 100%;
+    height: 5px;
+    background: #fff;
+}
+
+.hamburger::before {
+    top: 10px;
+}
+
+.hamburger::after {
+    top: 25px;
+}
+
+#toggle-menu {
+    display: none;
+}
+
+
+.menu ul {
```

Comment:
Przypominam tylko, aby stylować po klasach - wtedy łatwiej jest zapanować nad całością czyli zamiast `.menu ul` to np. `.submenu` lub podobnie.